lua/neoclip/fzf.lua

```diff
@@ -114,7 +114,7 @@ local function neoclip(register_names)
     end
     local actions = make_actions(register_names)
     require('fzf-lua').fzf_exec(fn, {
-      prompt = 'Prompt❯ ',
+      prompt = settings.prompt and settings.prompt or 'Prompt❯ ',
```

Comment:
```suggestion
      prompt = settings.prompt or 'Prompt❯ ',
```
shouldn't this be sufficient?

Also, would it make sense if prompt title also use `make_prompt_title` as telescope does?

Reply:
I don't think so. Telescope has both a prompt_title and a prompt_prefix. While fzf-lua has nothing like telescope's prompt_title, the "prompt_prefix" means the same thing as "prompt" for fzf (the characters before where someone can type).